### PR TITLE
Install mit postgre sql

### PR DESCRIPTION
**Pull request by thzanger**
The installation with PostgreSQL missed resetroot and newway to load epsg codes.

### CODE REVIEW

File: de/installation/installation_windows.rst [hunk 1]
```diff
@@ -189,6 +189,17 @@ Weitere Schritte unter:  `Mapbender Quickstart Dokument <../quickstart.html>`_.
...
 
-Das Symfony Welcome Script config.php öffnen. Das Skript prüft, ob alle notwendigen Komponenten installiert wurden und ob die Konfiguration erfolgte. Sofern noch Probleme vorliegen, sollten diese behoben werden.
- 
-* http://localhost/mapbender/config.php
+Überprüfen Sie in der Konfiguration, ob bestimmte Abhängigkeiten nicht erfüllt werden mit:
+
+
+.. code-block:: text
+
+    app/console mapbender:config:check
```
Comment (rschwammborn) [inline de/installation/installation_windows.rst:197]: Maybe it's useful to include `php.exe` to the command (like this: `php.exe app/console mapbender:config:check`)? Just for consistency...

File: en/installation/installation_windows.rst [hunk 2]
```diff
@@ -187,6 +188,14 @@ More information at:  `Mapbender Quickstart Document <../en/quickstart.html>`_.
...
 * http://localhost/mapbender/
 
-Open the Symfony Welcome Script config.php. The script will check if all required components are installed.
 
-* http://localhost/mapbender/config.php
+Check the system configuration and Mapbender requirements with:
+
+.. code-block:: text
+
+    app/console mapbender:config:check
```
Comment (rschwammborn) [inline en/installation/installation_windows.rst:196]: ...same suggestion as in the German version of the file (`php.exe`).

**Review by rschwammborn — COMMENTED** [review]
The changes are good. I made some small in-line suggestions in the files (which are not absolutely necessary to apply, but would imo be nice to have).

**Comment by eschuettler** [comment]
The line 'app/console fom:user:resetroot' has been deleted, because it was redundant information. In the documentation (where the information is needed) is a [link](https://doc.mapbender.org/en/installation/configuration.html#creating-the-administrative-user).
The link works in the German version, but I noticed that it won't work in the English version.

**Review by eschuettler — CHANGES_REQUESTED** [review]
The line 'app/console fom:user:resetroot' has been deleted, because it was redundant information. In the documentation (where the information is needed) is a link.
The link works in the German version, but I noticed that it won't work in the English version.

**Review by eschuettler — APPROVED** [review]

**Pull request merged**